### PR TITLE
Fix "not sequenced" in oncoprint

### PR DESCRIPTION
**Pull request by adamabeshouse**
Update oncoprint to use new system of coverage information.

image

image

image

image

non-screenshot E2E tests passing on local using `cbioportal/master`

### CODE REVIEW

**Comment by jjgao** [comment]
@adamabeshouse thanks. Looks good. Maybe also highlight profiles that were not profiled in the tooltip.

**Comment by pieterlukasse** [comment]
I am interested in reviewing this. @adamabeshouse can you please add some screenshots of before vs after?

**Comment by pieterlukasse** [comment]
@jjgao should we consider a sample with alteration data to be "profiled", irrespective of whether it has panel information or not?

File: end-to-end-tests/specs/home.spec.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ describe('homepage', function() {
         });
     }
 
-    it('it should have 27 (small test db), 29 (public test db) or 31 studies (production) in list', function () {
+    it.only('it should have 27 (small test db), 29 (public test db) or 31 studies (production) in list', function () {
```
Comment (inodb) [inline end-to-end-tests/specs/home.spec.js]: let's revert this before merging

Comment (adamabeshouse) [reply end-to-end-tests/specs/home.spec.js]: good catch

**Comment by jjgao** [comment]
@pieterlukasse I was thinking of that too. But since we are pull samples from sample_profile table, we should cover all samples in a profile already.

**Comment by jjgao** [comment]
@pieterlukasse `_sequenced` case list was used for mutation for now. I have asked Ben to fix that so all sequenced samples will be in `sample_profile` table.

AFAIR, data in `sample_profile` is complete for other types of data including cna.

**Comment by pieterlukasse** [comment]
@jjgao @n1zea144 can you please point to the PR where the `_sequenced` change is made? This PR would have to ensure the `_sequenced` case list input file is used to populate the `sample_profile` table, right. This would mean nothing has to change in the validation scripts, correct?

**Pull request merged**